openfisca_uk/parameters/demographic/population_estimate.yaml

```diff
@@ -0,0 +1,37 @@
+description: Population estimates for UK nations
+ENGLAND:
+  metadata:
+    interpolation:
+      interval: month
+  values:
+    2018-01-01: 56_000_000
+    2023-01-01: 57_600_000
+    2028-01-01: 58_800_000
+    2033-01-01: 59_800_000
+NORTHERN_IRELAND:
+  metadata:
+    interpolation:
+      interval: month
```

Comment:
Can we make this quarterly to align with the UC rollout?

Reply:
Actually that'd make it significantly more complex - the current implementation makes use of OpenFisca's periods (day, month, year). We could, but we'd need to add `quarter` as a period.